recipes/cyclus/run_test.sh

```diff
@@ -7,6 +7,14 @@ if [ -z "$CYCLUS_NUC_DATA" ]; then
   export CYCLUS_NUC_DATA="${PREFIX}/share/cyclus/cyclus_nuc_data.h5"
 fi
 
+UNAME="$(uname)"
+if [ "${UNAME}" == "Darwin" ]; then
+  export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${PREFIX}/lib:${PREFIX}/lib/cyclus"
```

Comment:
Could we please drop this? The fallback library (used below) should be sufficient and should be the preferred way to go.